src/components/avatar/Avatar.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import { ArrowDownIcon } from 'src/icons/ArrowDownIcon';
+
+import { DynamicIconAvatar } from './DynamicIconAvatar';
+import { IconAvatar } from './IconAvatar';
+import { ImageAvatar } from './ImageAvatar';
+import { UserAvatar } from './UserAvatar';
+
+describe('Avatar DOM snapshot tests', () => {
+  it('UserAvatar matches snapshot', () => {
+    const tree = renderer
+      .create(<UserAvatar size="lg" shape="round" />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
If I understand correctly,`it` and `test` are the same. But would it be confused if we use both in same project 🤔 I wonder if we should only use just either `it` or `test` function throughout our application or not 🤔 
```suggestion
  test('UserAvatar matches snapshot', () => {
    const tree = renderer
      .create(<UserAvatar size="lg" shape="round" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
```

Reply:
Whoops, not entirely sure why I used `it`. Must have been habit or something...